index.bs

```diff
@@ -1070,6 +1071,21 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
         1. [=set/Append=] |fakeReport| to the [=event-level report cache=].
     1. If |source|'s [=attribution source/randomized response=] is not [=set/is empty|empty=],
         then set |source|'s [=attribution source/event-level attributable=] value to false.
+    1. [=map/iterate|For each=] |destination| in [=source=]'s [=attribution source/attribution destinations=]:
```

Comment:
It is notable that this differs from the unnoised attribution rate-limits, which only count the "effective" destination. At the very least, we should add a note saying that the difference is intentional because at this time, we cannot say which destination should get the "credit" for the fake report.

However, it feels like something we should double check for the multi-destination design in general. cc @badih.

Reply:
Badih confirmed offline that this still provides the two-sided DP for user x source site x destination x reporting origin x time window privacy unit, so you're good to go. Can we include a note for why we need to do this? Something like 

```
Note: because a fake report does not have a "real" effective destination, we need to subtract from the privacy budget of all possible destinations.
```

Reply:
Updated the note, thanks!